Containerfile

```diff
@@ -18,4 +20,11 @@ RUN pip3 install -r requirements.txt --no-deps --no-cache-dir --require-hashes &
     # the git folder is only needed to determine the package version
     rm -rf .git
 
+WORKDIR /src/js-deps
+RUN npm install && \
+    ln -s "${PWD}/node_modules/.bin/corepack" /usr/local/bin/corepack && \
+    corepack enable && \
+    corepack prepare yarn@3.6.3 --activate && \
```

Comment:
We don't really need to prepare a specific yarn version, `corepack enable yarn` should be enough

And seeing the checksum-less `yarn` that we will be executing makes me sad :smiling_face_with_tear: 
(what's worse, it will be coming from the user's `packageManager` field in package.json)

I think I'll file another investigation to figure out if that's really a good idea or how to do it better

Reply:
Also a note on the commit message: the main point is not really installing yarn 3, but enabling Corepack's yarn shim. We will then dynamically fetch any yarn x.y.z defined by the user

Reply:
> We don't really need to prepare a specific yarn version

Ack. Is there any case where we would fallback to a globally installed yarn? The reason I pinned a specific version was because corepack appears to install the latest yarn 1.x if you don't specifically activate a different version.